<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Frederick Boehm">
<meta name="dcterms.date" content="2023-06-13">

<title>Testing pleiotropy in multiparental populations</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="utsw-mouse-slides_files/libs/clipboard/clipboard.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/quarto.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/popper.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/anchor.min.js"></script>
<link href="utsw-mouse-slides_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="utsw-mouse-slides_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="utsw-mouse-slides_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="utsw-mouse-slides_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="utsw-mouse-slides_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Testing pleiotropy in multiparental populations</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Frederick Boehm </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">June 13, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="multiparental-populations" class="level1">
<h1>Multiparental populations</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="utsw-mouse-slides_files/figure-html/mppfig-1.png" class="img-fluid" width="700"></p>
</div>
</div>
<p>???</p>
<ul>
<li>start with 8 inbred founder lines<br>
</li>
<li>inbred lines are color-coded<br>
</li>
<li>each subject represented by one homologous chromosome pair<br>
</li>
<li>over successive generations of outbreeding, subjects have smaller and smaller contiguous regions from a single inbred line</li>
</ul>
<hr>
</section>
<section id="background" class="level1">
<h1>Background</h1>
<ul>
<li><p>10,000+ traits with RNA sequencing and mass spectrometry</p></li>
<li><p>Quantitative trait locus mapping identifies genetic loci that affect measurable traits</p></li>
<li><p>Multiparental populations offer high-resolution QTL mapping</p></li>
<li><p>New analysis tools, such as a pleiotropy test for multiparental populations, are needed</p></li>
</ul>
<p>???</p>
<ul>
<li>Together, high-dimensional traits and multiparental populations can advance complex trait genetics</li>
</ul>
<hr>
</section>
<section id="benefits-of-a-new-pleiotropy-test" class="level1">
<h1>Benefits of a new pleiotropy test</h1>
<ul>
<li><p>Insights into genetic architecture</p></li>
<li><p>Tool for expression trait hotspot dissection</p></li>
<li><p>Complements mediation analysis</p></li>
</ul>
<p>???</p>
<p>When trying to identify an intermediate, a pleiotropy test limits the number of mediation analyses by identifying traits that share a QTL</p>
<hr>
</section>
<section id="jiang-and-zeng-1995-test" class="level1">
<h1>Jiang and Zeng (1995) test</h1>
<ul>
<li><p>Two-parent crosses</p></li>
<li><p>Applies to two traits that co-map</p></li>
<li><p><span class="math inline">\(H_0\)</span>: Pleiotropy</p></li>
<li><p><span class="math inline">\(H_A\)</span>: Two separate QTL</p></li>
</ul>
<hr>
</section>
<section id="jiang-and-zeng-1995-test-1" class="level1">
<h1>Jiang and Zeng (1995) test</h1>
<ul>
<li><p>Perform a two-dimensional two-QTL scan</p>
<ul>
<li><p><span class="math inline">\(vec(Y) = Xvec(B) + vec(E)\)</span></p></li>
<li><p>Calculate likelihood at each ordered pair of positions</p></li>
</ul></li>
<li><p>Calculate likelihood ratio test statistic</p></li>
</ul>
<p>???</p>
<ul>
<li><p>Y: 2 traits<br>
</p></li>
<li><p>X: genotype probs for 2 markers<br>
</p></li>
<li><p>B: founder allele effects<br>
</p></li>
<li><p>E: random errors, iid within each phenotype<br>
</p></li>
<li><p>vec: stack columns</p></li>
<li><p>2d scan involves all ordered pairs of markers</p></li>
</ul>
<hr>
</section>
<section id="challenges-in-multiparental-populations" class="level1">
<h1>Challenges in multiparental populations</h1>
<ul>
<li>Complex patterns of relatedness</li>
</ul>
<p>.soln[Multivariate random effects]</p>
<ul>
<li>Multiple founder lines</li>
</ul>
<p>.soln[Fixed effect for each founder allele]</p>
<p>.pull-left[<img src="figs/cc_founders.png" width="200" height="200">]</p>
<p>.pull-right[.small[Photo by UNC Computational Genetics]]</p>
<p>???</p>
<p>Two challenges occur at the point of relating phenotypes to genotypes</p>
<hr>
</section>
<section id="test-procedure" class="level1">
<h1>Test procedure</h1>
<ul>
<li><p>Perform a two-dimensional two-QTL scan</p>
<ul>
<li><p><span class="math inline">\(vec(Y) = Xvec(B) + vec(G) + vec(E)\)</span></p></li>
<li><p>Calculate likelihood at each ordered pair of positions</p></li>
</ul></li>
<li><p>Calculate likelihood ratio test statistic</p></li>
</ul>
<p>???</p>
<ul>
<li><p><span class="math inline">\(G\)</span>: Polygenic random effects that depend on the kinship matrix</p></li>
<li><p><span class="math inline">\(E\)</span>: Random errors</p></li>
</ul>
<hr>
</section>
<section id="test-procedure-1" class="level1">
<h1>Test procedure</h1>
<ul>
<li><p>Test statistic: <span class="math display">\[- \log_{10} \frac{\max (\text{likelihood under pleiotropy})}{\max (\text{likelihood for separate QTL})}\]</span></p></li>
<li><p>Parametric bootstrap to get a <span class="math inline">\(p\)</span>-value</p></li>
</ul>
<p>???</p>
<p>We fit the bivariate QTL models at all points on the two-dimensional grid. With the likelihoods of those model fits, we calculate a likelihood ratio test statistic.</p>
<p>Approximate the null distribution with the distribution of bootstrap test statistics</p>
<hr>
</section>
<section id="application" class="level1">
<h1>Application</h1>
<ul>
<li><p>Logan et al.&nbsp;(2013) and Recla et al.&nbsp;(2014) studied 261 Diversity Outbred mice</p></li>
<li><p>Measured about two dozen behavioral traits</p></li>
<li><p>Two traits map to Chr 8:</p>
<ul>
<li><p>“hot plate latency” (57 cM)</p></li>
<li><p>“percent time in light” (55 cM)</p></li>
</ul></li>
</ul>
<p>???</p>
<p>We applied our test to freely available data from the Chesler laboratory.</p>
<p>Do they share a single QTL?</p>
<p>We examined these two traits because they map to the same region</p>
<p>Hot plate latency is a measure of thermal pain sensitivity</p>
<p>Percent time in light is related to measures of anxiety</p>
<hr>
</section>
<section id="percent-time-in-light" class="level1">
<h1>Percent time in light</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="figs/percent-time-light-coef-lod-Chr8.svg" class="img-fluid" width="650"></p>
</div>
</div>
<p>???</p>
<ul>
<li>X axis is Chr 8 position</li>
<li>Y axis is allele effect (on the trait)</li>
<li>8 lines per figure: one for each founder line</li>
</ul>
<hr>
</section>
<section id="hot-plate-latency" class="level1">
<h1>Hot plate latency</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="figs/hot-plate-latency-coef-lod-Chr8.svg" class="img-fluid" width="650"></p>
</div>
</div>
<p>???</p>
<ul>
<li>Distinct allele effects patterns suggest separate QTL</li>
</ul>
<hr>
</section>
<section id="lod-definitions" class="level1">
<h1>LOD definitions</h1>
<ul>
<li><p><span class="math display">\[LOD(\lambda_1, \lambda_2) = ll_{10}(\lambda_1, \lambda_2) - \max_{\lambda} ll_{10}(\lambda, \lambda)\]</span></p></li>
<li><p><span class="math display">\[\text{profile LOD}_{\text{trait 1}}(\lambda_1) = \max_{\lambda_2}LOD(\lambda_1, \lambda_2)\]</span></p></li>
<li><p><span class="math display">\[LOD_p(\lambda) = ll_{10}(\lambda, \lambda) - \max_{\lambda} ll_{10}(\lambda, \lambda)\]</span></p></li>
</ul>
<hr>
</section>
<section id="profile-lod" class="level1">
<h1>Profile LOD</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="figs/profile.svg" class="img-fluid" width="650"></p>
</div>
</div>
<p>???</p>
<p>The x axis is Chr 8 position</p>
<p>The y axis is LOD</p>
<p>There are three “traces”: one for each trait and one for the pleiotropy hypothesis.</p>
<p>The pleiotropy trace derives from the likelihoods for the pleiotropy models (ie, the diagonal on the grid)</p>
<p>The “hot plate latency” profile LOD trace reveals a broad peak from 55 to 65 cM</p>
<p>The “percent time in light” profile LOD trace displays a sharp peak atop a broad region with higher LOD scores.</p>
<p>The triangles correspond to the univariate QTL peak positions, while the circles are the profile LOD peak positions.</p>
<p>The two profile LOD traces achieve the same maximum value, which is the test statistic value.</p>
<hr>
</section>
<section id="test-results" class="level1">
<h1>Test results</h1>
<ul>
<li><p><span class="math inline">\(\log_{10} \Lambda = 1.2\)</span></p></li>
<li><p><span class="math inline">\(p = 0.11\)</span> (1000 bootstrap samples)</p></li>
</ul>
<p>???</p>
<ul>
<li><p>Weak evidence for two separate QTL</p>
<ul>
<li><p>One QTL affects “distance traveled in light”</p></li>
<li><p>Second QTL affects “hot plate latency”</p></li>
</ul></li>
</ul>
</section>
<section id="data" class="level1">
<h1>Data</h1>
<ul>
<li><p>Keller et al.&nbsp;(2018) measured pancreatic islet gene expression levels in 378 Diversity Outbred mice</p></li>
<li><p>80 local expression QTL were identified in a 20-Mb region of Chr 19</p>
<ul>
<li>local: expression trait QTL is near gene position</li>
</ul></li>
<li><p>Assume that a local expression QTL affects only one local expression trait</p></li>
</ul>
<hr>
</section>
<section id="design" class="level1">
<h1>Design</h1>
<ul>
<li><p>Examine power with known expression trait QTL locations</p>
<ul>
<li>Interlocus distance<br>
</li>
<li>Univariate LOD</li>
</ul></li>
<li><p>4 strong, centrally located, local eQTL chosen as “anchor” traits</p>
<ul>
<li><em>Asah2, Lipo1, Lipo2, 4933413C19Rik</em></li>
</ul></li>
<li><p>Pairwise pleiotropy tests each involving one anchor and one of 79 other traits</p></li>
</ul>
<hr>
</section>
<section id="interlocus-distance" class="level1">
<h1>Interlocus distance</h1>
<hr>
</section>
<section id="interlocus-distance-1" class="level1">
<h1>Interlocus distance</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="figs/lrt-v-middle-of-gene-slides.svg" height="500"></p>
</div>
</div>
<hr>
</section>
<section id="univariate-lod" class="level1">
<h1>Univariate LOD</h1>
<div class="cell">
<div class="cell-output-display">
<p><img src="figs/lrt-v-univariate-lod-slides.svg" height="500"></p>
</div>
</div>
<hr>
</section>
<section id="conclusions" class="level1">
<h1>Conclusions</h1>
<ul>
<li><span class="math inline">\(\uparrow\)</span> Pleiotropy test statistics
<ul>
<li><span class="math inline">\(\uparrow\)</span> Interlocus distance</li>
<li><span class="math inline">\(\uparrow\)</span> Univariate LOD</li>
</ul></li>
</ul>
</section>
<section id="qtl2pleio-r-package" class="level1">
<h1>qtl2pleio R package</h1>
<ul>
<li><p>Functions for <span class="math inline">\(d\)</span>-variate, <span class="math inline">\(d\)</span>-QTL scan &amp; profile LOD plots</p></li>
<li><p>Uses C++ for matrix calculations (via Rcpp and RcppEigen)</p></li>
<li><p>Uses <code>gemma2</code> R implementation of GEMMA EM algorithm for multivariate random effects</p></li>
<li><p>Unit tests, vignettes, and version control</p></li>
</ul>
<p>???</p>
<ul>
<li><p>unit tests to ensure accurate calculations</p></li>
<li><p>vignettes on analyzing a pair of traits &amp; on cluster computing for bootstrap test</p></li>
</ul>
<hr>
</section>
<section id="summary" class="level1">
<h1>Summary</h1>
<ol type="1">
<li>Background<br>
</li>
<li>Methods<br>
</li>
<li>Applications
<ol type="a">
<li>Pleiotropy testing and mediation analysis<br>
</li>
<li>Power in pleiotropy testing<br>
</li>
<li>Microbiome case study<br>
</li>
</ol></li>
<li>Software<br>
</li>
<li>Conclusions</li>
</ol>
<hr>
</section>
<section id="references" class="level1">
<h1>References</h1>
<p>Jiang, C. and Z. Zeng (1995). “Multiple trait analysis of genetic mapping for quantitative trait loci.” In: <em>Genetics</em> 140.3, pp. 1111-1127.</p>
<p>Keller, M. P., D. M. Gatti, et al.&nbsp;(2018). “Genetic Drivers of Pancreatic Islet Function”. In: <em>Genetics</em>, pp.&nbsp;genetics-300864.</p>
<p>Logan, R. W., R. F. Robledo, et al.&nbsp;(2013). “High-precision genetic mapping of behavioral traits in the diversity outbred mouse population”. In: <em>Genes, Brain and Behavior</em> 12.4, pp.&nbsp;424-437.</p>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>