---
title: "Testing Pleiotropy vs. Separate QTL in Multiparental Populations"
author: Frederick Boehm
date: 'June 13, 2023'
date-format: full
bibliography: research.bib
toc: true
toc-depth: 1
toc-title: "Overview"
engine: knitr
keep-md: true
filters: [bg_style.lua]
format:
    revealjs:
        theme: "white"
        footer: "Frederick Boehm (frederick.boehm@gmail.com)  https://fboehm.us/"
        logo: "figs/vitruvian-mouse-eleni.png"
        progress: true
        slide-tone: true
        chalkboard: true
        auto-stretch: false
        transition: "slide"
        slide-number: "c/t"
---




# Quantitative Trait Locus (QTL) Mapping {.center} 

```{css, echo=FALSE}
.center h1 {
  text-align: center;
}
.inverse h1 {
   color: white;
}

.reveal .slide-logo {
  height: 150px !important;
  width: 150px!important;
  max-width: unset !important;
  max-height: unset !important;
}
```




```{scss}
.soln {
    color: red; 
    font-style: italic;
}
.red { color: red; }

.small{ font-size: 16px;}

.tiny{ font-size: 10px;}
```


```{r setup, include=FALSE}
options(htmltools.dir.version = FALSE)
library(tidyverse)
library(ggalt)
library(knitr)
ratio <- "4:3"
fig_height <- ifelse(ratio=='4:3', 10, 7.5)
fig_width <- 15
curves_height <- ifelse(ratio=='4:3', 580, 540)
curves_width <- ifelse(ratio=='4:3', 800, 1080)
knitr::opts_chunk$set(#dev="svg", 
                      echo=FALSE, 
                      #results="hide",
                      #fig.height=fig_height,
                      #fig.width=fig_width,
                      warning=FALSE, message=FALSE)
```





## Motivation

- Variations in quantitative traits are due to genetic and environmental factors  
- Quantitative trait loci: number, locations, effects, identities  
- Biological insights: new drugs, improve breeding, natural selection


::: {.notes}

- Knowledge of the number, locations, effects, and identities of these factors can contribute to new biological insights  
    - new drugs  
    - improve breeding in agriculture  
    - better understanding of natural selection   

:::

---

## What is QTL mapping?

- A statistical method to identify genetic factors, or loci, that affect a quantitative trait  
- Consider each marker, one at a time, and test whether it affects the trait  

---

## Why use model organisms?

- We could study hypertension in humans  
- But human studies have limitations:  
    - Cost  
    - Inability to control subjects' life histories and environments  
    - Confounding   

---

## Why use model organisms?

- Shared biology with humans  
- Shorter generation time   
- Large numbers of progeny    
- Controlled environments   
- Control genetic background  
- Valid causal inferences  

---

## Limitations of Model Organisms

- Relevance to human biology  
- QTL identification $\neq$ Gene identification

::: {.notes}

- A single QTL may contain dozens or even hundreds of genes


:::




---

## QTL mapping in two-parent crosses


```{r}
##############################
# The intercross
##############################

library(broman)
library(simcross)

color <- brocolors("crayons")[c("Cornflower", "Blush")]
source(here::here("R/colors.R"))
#bgcolor <- "white"
#par(fg="white", bty="n", bg=bgcolor)
#par(fg = "white", bty = "n" )
par(bty="n")
par(mar=rep(0, 4))

plot(0,0,xlim=c(0,864),ylim=c(0,480),xaxt="n",yaxt="n",xlab="",ylab="",type="n")
rect(c(300,328),c(480,480),c(310,338),c(385,385),col=color[1],border=color[1], lend=1, ljoin=1)
rect(c(526,554),c(480,480),c(536,564),c(385,385),col=color[2],border=color[2], lend=1, ljoin=1)

points(432,440,pch=4,cex=2.5,lwd=2)
segments(432,400,432,340,lwd=2)
segments(319,340,545,340,lwd=2)
arrows(c(319,545),c(340,340),c(319,545),c(300,300),lwd=2,len=0.1)

text(300-25,(480+385)/2,expression(P[1]),cex=2,adj=c(1,0.5))
text(564+25,(480+385)/2,expression(P[2]),cex=2,adj=c(0,0.5))

rect(300,287,310,192,col=color[1],border=color[1], lend=1, ljoin=1)
rect(328,287,338,192,col=color[2],border=color[2], lend=1, ljoin=1)
rect(526,287,536,192,col=color[1],border=color[1], lend=1, ljoin=1)
rect(554,287,564,192,col=color[2],border=color[2], lend=1, ljoin=1)

points(432,247,pch=4,cex=2.5,lwd=2)
segments(432,207,432,147,lwd=2)
segments(57,147,849,147,lwd=2)
arrows(seq(57,849,by=88),rep(147,10),seq(57,849,by=88),rep(107,10),lwd=2,len=0.1)

text(300-25,(287+192)/2,expression(F[1]),cex=2,adj=c(1,0.5))
text(564+25,(287+192)/2,expression(F[1]),cex=2,adj=c(0,0.5))

f1 <- create_parent(100,c(1,2))
set.seed(994)
f2 <- vector("list",10)
for(i in 1:10) f2[[i]] <- cross(f1,f1,m=10)

xloc <- 38
mult <- 95/max(f2[[1]]$mat$locations)
for(i in 1:10) {
    rect(xloc,0,xloc+10,95,   col=color[1],border=color[1], lend=1, ljoin=1)
    rect(xloc+28,0,xloc+38,95,col=color[1],border=color[1], lend=1, ljoin=1)

    f2m <- f2[[i]]$mat
    f2m$locations <- c(0, f2m$locations)
    for(j in 2:length(f2m$locations)) {
        if(f2m$alleles[j-1]==2) {
            rect(xloc,f2m$locations[j]*mult,xloc+10,f2m$locations[j-1]*mult,col=color[2],border=color[2], lend=1, ljoin=1)
        }
    }
    f2p <- f2[[i]]$pat
    f2p$locations <- c(0, f2p$locations)
    for(j in 2:length(f2p$locations)) {
        if(f2p$alleles[j-1]==2) {
            rect(xloc+28,f2p$locations[j]*mult,xloc+38,f2p$locations[j-1]*mult,col=color[2],border=color[2], lend=1, ljoin=1)
        }
    }
    xloc <- xloc+38+50
}
text(38-25,95/2,expression(F[2]),cex=2,adj=c(1,0.5))
```

::: {.notes}

- Cross two inbred lines to produce a F1 population  
- Cross F1 subjects to produce a F2 population    
- Genotype F2 subjects at many markers  
- Phenotype F2 subjects for the trait of interest  
- Test each marker for association with the trait  

:::




# QTL Mapping in Multiparental Populations




## Multiparental populations

```{r mppfig, echo = FALSE, out.width = "700px", fig.height=fig_height, fig.width = fig_width}
source(here::here("R/hs_fig.R"))
# https://github.com/kbroman/Talk_JAX2018/blob/master/R/hs_fig.R
```



::: {.notes}

- start with 8 inbred founder lines  
- inbred lines are color-coded  
- each subject represented by one homologous chromosome pair  
- over successive generations of outbreeding, subjects have smaller and smaller contiguous regions from a single inbred line  
:::


---

## Diversity Outbred Mice


![](figs/DO-mating-design.png){.absolute bottom=30 left=200 width=700 height=550}


::: {.notes}

- 8 inbred founder lines  
- Funnel design to incorporate DNA from all 8 founders  
- 30+ generations of outbreeding  

:::

---

## Motivation

- 10,000+ traits with RNA sequencing and mass spectrometry 

- Multiparental populations offer high-resolution QTL mapping

- New analysis tools, such as a pleiotropy test for multiparental populations, are needed 

![](figs/mouse_on_chips.png){.absolute bottom=30 left=450 width="300" height="250"}


::: {.notes}

- Together, high-dimensional traits and multiparental populations can advance complex trait genetics

:::




# A New Pleiotropy Test




## Benefits of a new pleiotropy test

- Insights into genetic architecture

- Tool for expression trait hotspot dissection

- Complements mediation analysis


::: {.notes}

When trying to identify an intermediate, a pleiotropy test limits the number of mediation analyses by identifying traits that share a QTL

:::






---

## @jiang1995multiple test

- Two-parent crosses    

- Applies to two traits that co-map     

- $H_0$: Pleiotropy    

- $H_A$: Two separate QTL    



---

## @jiang1995multiple test 


- Perform a two-dimensional two-QTL scan     

- $vec(Y) = Xvec(B) + vec(E)$     
    
 
```{r, grid}
#| fig.height=4, fig.width=4

# Create an empty plot with the desired range
limits <- c(0.5, 5.5)
plot(x = 1:5, y = 1:5, type = "n", xlab = "Genomic Position (Trait 1)", ylab = "Genomic Position (Trait 2)", xlim = limits, ylim = limits)

# Define a vector of colors
colors <- rep("red", 25)
green_indices <- c(1, 7, 13, 19, 25)
colors[green_indices] <- "green"
# Add points to represent the grid with different colors
points(rep(1:5, 5), rep(1:5, each = 5), pch = 19, cex = 2, col = colors)
```



::: {.notes}

- Y: n by 2 matrix for 2 traits  
- X: 2n by 2 block diagonal matrix genotype probabilities for 2 markers  
- B: founder allele effects  
- E: random errors, iid within each phenotype  
- vec: stack columns

- 2d scan involves all ordered pairs of markers

- Calculate likelihood at each ordered pair of positions    

- Calculate likelihood ratio test statistic     

:::


---

## Challenges in multiparental populations

- Complex patterns of relatedness    

::: {.soln}
Multivariate random effects
:::

- Multiple founder lines

::: {.soln}
Fixed effect for each founder allele
:::





::: {.notes}

Two challenges occur at the point of relating phenotypes to genotypes
:::


---

## Test procedure

- Perform a two-dimensional two-QTL scan     

- $vec(Y) = Xvec(B) + vec(G) + vec(E)$     
    
```{r, grid_again}
#| fig.height=4, fig.width=4
<<grid>>
```

::: {.notes}

- $G$: Polygenic random effects that depend on the kinship matrix  
    
- $E$: Random errors

- Calculate likelihood at each ordered pair of positions    

- Calculate likelihood ratio test statistic     

:::  


---

## Test procedure

- Test statistic: $$- \log_{10} \frac{\max (\text{likelihood under pleiotropy})}{\max (\text{likelihood for separate QTL})}$$    

- Parametric bootstrap to get a $p$-value    


::: {.notes}

We fit the bivariate QTL models at all points on the two-dimensional grid. With the likelihoods of those model fits, we calculate a likelihood ratio test statistic.

Approximate the null distribution with the distribution of bootstrap test statistics

:::



# Chesler, et al. Data Analysis  


## @logan2013high & @recla2014precise

- 261 Diversity Outbred mice    

- Measured about two dozen behavioral traits

![](figs/lightdarkbox.jpg){.absolute bottom=30 left=50 width="350" height="300"}

![](figs/hotplate-2.jpg){.absolute bottom=30 right=50 width="350" height="300"}







::: {.notes}

- Two traits map to Chr 8:

    - "hot plate latency" (57 cM)    

    - "percent time in light" (55 cM)

We applied our test to freely available data from the Chesler laboratory.


Do the two traits share a single QTL? 

We examined these two traits because they map to the same region

Hot plate latency is a measure of thermal pain sensitivity

Percent time in light is related to measures of anxiety

:::

---

## Data Structures

- $261$ subjects by $8$ founder lines matrix per marker  
 
- $261$ subjects by $261$ subjects (LOCO) kinship matrix (per chromosome)

- $261$ subjects by $2$ traits matrix


::: {.notes}

- Each chromosome's genetics data is a separate array, with one matrix per marker

:::

---

## Percent time in light


```{r percentlight, echo = FALSE, out.width = "650px"}
include_graphics("figs/percent-time-light-coef-lod-Chr8.svg")
```


::: {.notes}

- X axis is Chr 8 position
- Y axis is allele effect (on the trait)
- 8 lines per figure: one for each founder line

:::


---

## Hot plate latency


```{r hotplate, echo = FALSE, out.width = "650px"}
include_graphics("figs/hot-plate-latency-coef-lod-Chr8.svg")
```



::: {.notes}

- Distinct allele effects patterns suggest separate QTL

:::

---

## Correlation between traits

```{r}
include_graphics("figs/scatter-1.png")
```

::: {.notes}

- Correlation coefficient estimate is $-0.15$

:::

---


## Profile LOD


```{r profiles, out.width = "650px", echo = FALSE}
include_graphics("figs/profile.svg", auto_pdf = FALSE)
```

::: {.notes}

The x axis is Chr 8 position

The y axis is LOD

There are three "traces": one for each trait and one for the pleiotropy hypothesis.

The pleiotropy trace derives from the likelihoods for the pleiotropy models (ie, the diagonal on the grid)

The "hot plate latency" profile LOD trace reveals a broad peak from 55 to 65 cM

The "percent time in light" profile LOD trace displays a sharp peak atop a broad region with higher LOD scores. 

The triangles correspond to the univariate QTL peak positions, while the circles are the profile LOD peak positions.

The two profile LOD traces achieve the same maximum value, which is the test statistic value. 

:::

---

## Test results


- $\log_{10} \Lambda = 1.2$

- $p = 0.11$ (1000 bootstrap samples)

::: {.notes}

- Weak evidence for two separate QTL

    - One QTL affects "distance traveled in light"    
    
    - Second QTL affects "hot plate latency"

:::


# Simulation Studies


## Assess Type I Error Rate

```{r}
include_graphics("figs/table-sims.png")
```

::: {.notes}

- 2 by 2 by 2 factorial design  
- 400 trait pairs per scenario  
- 400 bootstrap samples per trait pair  


:::

---

## Assessing Power

```{r}
include_graphics("figs/power-curves.svg")
```

::: {.notes}

- 2 by 2 by 5 factorial design  
- 3 factors: allele effects difference, allele effects partitioning, and inter-locus distance  
- Simulated 400 trait pairs per scenario
- 400 bootstrap samples per trait pair



:::


# `qtl2pleio` R package



## `qtl2pleio` R package


```{r}
include_graphics("figs/qtl2pleio-github-screenshot.png")
```



::: {.notes}

- Functions for $d$-variate, $d$-QTL scan & profile LOD plots  

- Uses C++ for matrix calculations (via Rcpp and RcppEigen)  

- Uses `gemma2` R implementation of GEMMA EM algorithm for multivariate random effects  

- Unit tests, vignettes, and version control  

- Open source development on GitHub  

- unit tests to ensure accurate calculations

- vignettes on analyzing a pair of traits & on cluster computing for bootstrap test
:::

---

## `qtl2pleio` R package 

- `qtl2pleio` R package: <https://fboehm.us/software/qtl2pleio/>

```{r}
include_graphics("figs/qtl2pleio-pkgdown-site-screenshot.png")
```

::: {.notes}

- Also available on CRAN  

:::





---


## Thank you!



---

## LOD definitions {visibility="uncounted"}

- $$LOD(\lambda_1, \lambda_2) = ll_{10}(\lambda_1, \lambda_2) - \max_{\lambda} ll_{10}(\lambda, \lambda)$$


- $$\text{profile LOD}_{\text{trait 1}}(\lambda_1) = \max_{\lambda_2}LOD(\lambda_1, \lambda_2)$$



- $$LOD_p(\lambda) = ll_{10}(\lambda, \lambda) - \max_{\lambda} ll_{10}(\lambda, \lambda)$$


---

## Parametric bootstrap test {visibility="uncounted"}

- Simulate data under the null hypothesis  
- Calculate test statistic for each simulated dataset  
- Determine proportion of simulated test statistics that exceed observed test statistic  


::: {.notes}


:::





---



## References {visibility="uncounted"}




