<!DOCTYPE html>
<html lang="en"><head>
<script src="utsw-mouse-slides_files/libs/tone/tone.js"></script>
<script src="utsw-mouse-slides_files/libs/clipboard/clipboard.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/tabby.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/popper.min.js"></script>
<script src="utsw-mouse-slides_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="utsw-mouse-slides_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="utsw-mouse-slides_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="utsw-mouse-slides_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.2.335">

  <meta name="author" content="Frederick Boehm">
  <title>Testing Pleiotropy vs.&nbsp;Separate QTL in Multiparental Populations</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="utsw-mouse-slides_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="utsw-mouse-slides_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <link rel="stylesheet" href="utsw-mouse-slides_files/libs/revealjs/dist/theme/quarto.css" id="theme">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-chalkboard/font-awesome/css/all.css" rel="stylesheet">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-chalkboard/style.css" rel="stylesheet">
  <link href="utsw-mouse-slides_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-captioned.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-captioned) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-captioned.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-captioned .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-captioned .callout-caption  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-captioned.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-captioned.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-caption {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-caption {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-caption {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-captioned .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-captioned .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-captioned) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-caption {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-caption {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-caption {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-caption {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-caption {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Testing Pleiotropy vs.&nbsp;Separate QTL in Multiparental Populations</h1>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Frederick Boehm 
</div>
</div>
</div>

  <p class="date">Tuesday, June 13, 2023</p>
</section><section id="TOC">
<nav role="doc-toc"> 
<h2 id="toc-title">Overview</h2>
<ul>
<li><a href="#/quantitative-trait-locus-qtl-mapping" id="/toc-quantitative-trait-locus-qtl-mapping">Quantitative Trait Locus (QTL) Mapping</a></li>
<li><a href="#/qtl-mapping-in-multiparental-populations" id="/toc-qtl-mapping-in-multiparental-populations">QTL Mapping in Multiparental Populations</a></li>
<li><a href="#/a-new-pleiotropy-test" id="/toc-a-new-pleiotropy-test">A New Pleiotropy Test</a></li>
<li><a href="#/chesler-et-al.-data-analysis" id="/toc-chesler-et-al.-data-analysis">Chesler, et al.&nbsp;Data Analysis</a></li>
<li><a href="#/simulation-studies" id="/toc-simulation-studies">Simulation Studies</a></li>
<li><a href="#/qtl2pleio-r-package" id="/toc-qtl2pleio-r-package"><code>qtl2pleio</code> R package</a></li>
</ul>
</nav>
</section>
<section>
<section id="quantitative-trait-locus-qtl-mapping" class="title-slide slide level1 center inverse" data-background-color="black">
<h1>Quantitative Trait Locus (QTL) Mapping</h1>
<div class="cell">
<style type="text/css">
.center h1 {
  text-align: center;
}
.inverse h1 {
   color: white;
}

.reveal .slide-logo {
  height: 150px !important;
  width: 150px!important;
  max-width: unset !important;
  max-height: unset !important;
}
</style>
</div>
<div class="cell">
<style type="text/css">
.soln{color:red;font-style:italic}.red{color:red}.small{font-size:16px}.tiny{font-size:10px}
</style>
</div>
</section>
<section id="motivation" class="slide level2">
<h2>Motivation</h2>
<ul>
<li>Variations in quantitative traits are due to genetic and environmental factors<br>
</li>
<li>Quantitative trait loci: number, locations, effects, identities<br>
</li>
<li>Biological insights: new drugs, improve breeding, natural selection</li>
</ul>
<aside class="notes">
<ul>
<li>Knowledge of the number, locations, effects, and identities of these factors can contribute to new biological insights
<ul>
<li>new drugs<br>
</li>
<li>improve breeding in agriculture<br>
</li>
<li>better understanding of natural selection</li>
</ul></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="what-is-qtl-mapping" class="slide level2">
<h2>What is QTL mapping?</h2>
<ul>
<li>A statistical method to identify genetic factors, or loci, that affect a quantitative trait<br>
</li>
<li>Consider each marker, one at a time, and test whether it affects the trait</li>
</ul>
</section>
<section id="why-use-model-organisms" class="slide level2">
<h2>Why use model organisms?</h2>
<ul>
<li>We could study hypertension in humans<br>
</li>
<li>But human studies have limitations:
<ul>
<li>Cost<br>
</li>
<li>Inability to control subjects’ life histories and environments<br>
</li>
<li>Confounding</li>
</ul></li>
</ul>
</section>
<section id="why-use-model-organisms-1" class="slide level2">
<h2>Why use model organisms?</h2>
<ul>
<li>Shared biology with humans<br>
</li>
<li>Shorter generation time<br>
</li>
<li>Large numbers of progeny<br>
</li>
<li>Controlled environments<br>
</li>
<li>Control genetic background<br>
</li>
<li>Valid causal inferences</li>
</ul>
</section>
<section id="limitations-of-model-organisms" class="slide level2">
<h2>Limitations of Model Organisms</h2>
<ul>
<li>Relevance to human biology<br>
</li>
<li>QTL identification <span class="math inline">\(\neq\)</span> Gene identification</li>
</ul>
<aside class="notes">
<ul>
<li>A single QTL may contain dozens or even hundreds of genes</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="qtl-mapping-in-two-parent-crosses" class="slide level2">
<h2>QTL mapping in two-parent crosses</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="utsw-mouse-slides_files/figure-revealjs/unnamed-chunk-3-1.png" width="960"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>Cross two inbred lines to produce a F1 population<br>
</li>
<li>Cross F1 subjects to produce a F2 population<br>
</li>
<li>Genotype F2 subjects at many markers<br>
</li>
<li>Phenotype F2 subjects for the trait of interest<br>
</li>
<li>Test each marker for association with the trait</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="qtl-mapping-in-multiparental-populations" class="title-slide slide level1 inverse center" data-background-color="black">
<h1>QTL Mapping in Multiparental Populations</h1>

</section>
<section id="multiparental-populations" class="slide level2">
<h2>Multiparental populations</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="utsw-mouse-slides_files/figure-revealjs/mppfig-1.png" width="700"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>start with 8 inbred founder lines<br>
</li>
<li>inbred lines are color-coded<br>
</li>
<li>each subject represented by one homologous chromosome pair<br>
</li>
<li>over successive generations of outbreeding, subjects have smaller and smaller contiguous regions from a single inbred line<br>
</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="diversity-outbred-mice" class="slide level2">
<h2>Diversity Outbred Mice</h2>
<p><img data-src="figs/DO-mating-design.png" class="absolute" style="left: 200px; bottom: 30px; width: 700px; height: 550px; "></p>
<aside class="notes">
<ul>
<li>8 inbred founder lines<br>
</li>
<li>Funnel design to incorporate DNA from all 8 founders<br>
</li>
<li>30+ generations of outbreeding</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="motivation-1" class="slide level2">
<h2>Motivation</h2>
<ul>
<li><p>10,000+ traits with RNA sequencing and mass spectrometry</p></li>
<li><p>Multiparental populations offer high-resolution QTL mapping</p></li>
<li><p>New analysis tools, such as a pleiotropy test for multiparental populations, are needed</p></li>
</ul>
<p><img data-src="figs/mouse_on_chips.png" class="absolute" style="left: 450px; bottom: 30px; width: 300px; height: 250px; "></p>
<aside class="notes">
<ul>
<li>Together, high-dimensional traits and multiparental populations can advance complex trait genetics</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="a-new-pleiotropy-test" class="title-slide slide level1 inverse center" data-background-color="black">
<h1>A New Pleiotropy Test</h1>

</section>
<section id="benefits-of-a-new-pleiotropy-test" class="slide level2">
<h2>Benefits of a new pleiotropy test</h2>
<ul>
<li><p>Insights into genetic architecture</p></li>
<li><p>Tool for expression trait hotspot dissection</p></li>
<li><p>Complements mediation analysis</p></li>
</ul>
<aside class="notes">
<p>When trying to identify an intermediate, a pleiotropy test limits the number of mediation analyses by identifying traits that share a QTL</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="jiang1995multiple-test" class="slide level2">
<h2><span class="citation" data-cites="jiang1995multiple">Jiang and Zeng (<a href="#/references" role="doc-biblioref" onclick="">1995</a>)</span> test</h2>
<ul>
<li><p>Two-parent crosses</p></li>
<li><p>Applies to two traits that co-map</p></li>
<li><p><span class="math inline">\(H_0\)</span>: Pleiotropy</p></li>
<li><p><span class="math inline">\(H_A\)</span>: Two separate QTL</p></li>
</ul>
</section>
<section id="jiang1995multiple-test-1" class="slide level2">
<h2><span class="citation" data-cites="jiang1995multiple">Jiang and Zeng (<a href="#/references" role="doc-biblioref" onclick="">1995</a>)</span> test</h2>
<ul>
<li><p>Perform a two-dimensional two-QTL scan</p></li>
<li><p><span class="math inline">\(vec(Y) = Xvec(B) + vec(E)\)</span></p></li>
</ul>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="utsw-mouse-slides_files/figure-revealjs/grid-1.png" width="384"></p>
</div>
</div>
<aside class="notes">
<ul>
<li><p>Y: n by 2 matrix for 2 traits<br>
</p></li>
<li><p>X: 2n by 2 block diagonal matrix genotype probabilities for 2 markers<br>
</p></li>
<li><p>B: founder allele effects<br>
</p></li>
<li><p>E: random errors, iid within each phenotype<br>
</p></li>
<li><p>vec: stack columns</p></li>
<li><p>2d scan involves all ordered pairs of markers</p></li>
<li><p>Calculate likelihood at each ordered pair of positions</p></li>
<li><p>Calculate likelihood ratio test statistic</p></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="challenges-in-multiparental-populations" class="slide level2">
<h2>Challenges in multiparental populations</h2>
<ul>
<li>Complex patterns of relatedness</li>
</ul>
<div class="soln">
<p>Multivariate random effects</p>
</div>
<ul>
<li>Multiple founder lines</li>
</ul>
<div class="soln">
<p>Fixed effect for each founder allele</p>
</div>
<aside class="notes">
<p>Two challenges occur at the point of relating phenotypes to genotypes</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="test-procedure" class="slide level2">
<h2>Test procedure</h2>
<ul>
<li><p>Perform a two-dimensional two-QTL scan</p></li>
<li><p><span class="math inline">\(vec(Y) = Xvec(B) + vec(G) + vec(E)\)</span></p></li>
</ul>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="utsw-mouse-slides_files/figure-revealjs/grid_again-1.png" width="384"></p>
</div>
</div>
<aside class="notes">
<ul>
<li><p><span class="math inline">\(G\)</span>: Polygenic random effects that depend on the kinship matrix</p></li>
<li><p><span class="math inline">\(E\)</span>: Random errors</p></li>
<li><p>Calculate likelihood at each ordered pair of positions</p></li>
<li><p>Calculate likelihood ratio test statistic</p></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="test-procedure-1" class="slide level2">
<h2>Test procedure</h2>
<ul>
<li><p>Test statistic: <span class="math display">\[- \log_{10} \frac{\max (\text{likelihood under pleiotropy})}{\max (\text{likelihood for separate QTL})}\]</span></p></li>
<li><p>Parametric bootstrap to get a <span class="math inline">\(p\)</span>-value</p></li>
</ul>
<aside class="notes">
<p>We fit the bivariate QTL models at all points on the two-dimensional grid. With the likelihoods of those model fits, we calculate a likelihood ratio test statistic.</p>
<p>Approximate the null distribution with the distribution of bootstrap test statistics</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="chesler-et-al.-data-analysis" class="title-slide slide level1 inverse center" data-background-color="black">
<h1>Chesler, et al.&nbsp;Data Analysis</h1>

</section>
<section id="logan2013high-recla2014precise" class="slide level2">
<h2><span class="citation" data-cites="logan2013high">Logan et al. (<a href="#/references" role="doc-biblioref" onclick="">2013</a>)</span> &amp; <span class="citation" data-cites="recla2014precise">Recla et al. (<a href="#/references" role="doc-biblioref" onclick="">2014</a>)</span></h2>
<ul>
<li><p>261 Diversity Outbred mice</p></li>
<li><p>Measured about two dozen behavioral traits</p></li>
</ul>
<p><img data-src="figs/lightdarkbox.jpg" class="absolute" style="left: 50px; bottom: 30px; width: 350px; height: 300px; "></p>
<p><img data-src="figs/hotplate-2.jpg" class="absolute" style="bottom: 30px; right: 50px; width: 350px; height: 300px; "></p>
<aside class="notes">
<ul>
<li><p>Two traits map to Chr 8:</p>
<ul>
<li><p>“hot plate latency” (57 cM)</p></li>
<li><p>“percent time in light” (55 cM)</p></li>
</ul></li>
</ul>
<p>We applied our test to freely available data from the Chesler laboratory.</p>
<p>Do the two traits share a single QTL?</p>
<p>We examined these two traits because they map to the same region</p>
<p>Hot plate latency is a measure of thermal pain sensitivity</p>
<p>Percent time in light is related to measures of anxiety</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="data-structures" class="slide level2">
<h2>Data Structures</h2>
<ul>
<li><p><span class="math inline">\(261\)</span> subjects by <span class="math inline">\(8\)</span> founder lines matrix per marker</p></li>
<li><p><span class="math inline">\(261\)</span> subjects by <span class="math inline">\(261\)</span> subjects (LOCO) kinship matrix (per chromosome)</p></li>
<li><p><span class="math inline">\(261\)</span> subjects by <span class="math inline">\(2\)</span> traits matrix</p></li>
</ul>
<aside class="notes">
<ul>
<li>Each chromosome’s genetics data is a separate array, with one matrix per marker</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="percent-time-in-light" class="slide level2">
<h2>Percent time in light</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/percent-time-light-coef-lod-Chr8.svg" width="650"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>X axis is Chr 8 position</li>
<li>Y axis is allele effect (on the trait)</li>
<li>8 lines per figure: one for each founder line</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="hot-plate-latency" class="slide level2">
<h2>Hot plate latency</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/hot-plate-latency-coef-lod-Chr8.svg" width="650"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>Distinct allele effects patterns suggest separate QTL</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="correlation-between-traits" class="slide level2">
<h2>Correlation between traits</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/scatter-1.png"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>Correlation coefficient estimate is <span class="math inline">\(-0.15\)</span></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="lod-definitions" class="slide level2">
<h2>LOD definitions</h2>
<ul>
<li><p><span class="math display">\[LOD(\lambda_1, \lambda_2) = ll_{10}(\lambda_1, \lambda_2) - \max_{\lambda} ll_{10}(\lambda, \lambda)\]</span></p></li>
<li><p><span class="math display">\[\text{profile LOD}_{\text{trait 1}}(\lambda_1) = \max_{\lambda_2}LOD(\lambda_1, \lambda_2)\]</span></p></li>
<li><p><span class="math display">\[LOD_p(\lambda) = ll_{10}(\lambda, \lambda) - \max_{\lambda} ll_{10}(\lambda, \lambda)\]</span></p></li>
</ul>
</section>
<section id="profile-lod" class="slide level2">
<h2>Profile LOD</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/profile.svg" width="650"></p>
</div>
</div>
<aside class="notes">
<p>The x axis is Chr 8 position</p>
<p>The y axis is LOD</p>
<p>There are three “traces”: one for each trait and one for the pleiotropy hypothesis.</p>
<p>The pleiotropy trace derives from the likelihoods for the pleiotropy models (ie, the diagonal on the grid)</p>
<p>The “hot plate latency” profile LOD trace reveals a broad peak from 55 to 65 cM</p>
<p>The “percent time in light” profile LOD trace displays a sharp peak atop a broad region with higher LOD scores.</p>
<p>The triangles correspond to the univariate QTL peak positions, while the circles are the profile LOD peak positions.</p>
<p>The two profile LOD traces achieve the same maximum value, which is the test statistic value.</p>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="test-results" class="slide level2">
<h2>Test results</h2>
<ul>
<li><p><span class="math inline">\(\log_{10} \Lambda = 1.2\)</span></p></li>
<li><p><span class="math inline">\(p = 0.11\)</span> (1000 bootstrap samples)</p></li>
</ul>
<aside class="notes">
<ul>
<li><p>Weak evidence for two separate QTL</p>
<ul>
<li><p>One QTL affects “distance traveled in light”</p></li>
<li><p>Second QTL affects “hot plate latency”</p></li>
</ul></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="parametric-bootstrap-test" class="slide level2">
<h2>Parametric bootstrap test</h2>
<ul>
<li>Simulate data under the null hypothesis<br>
</li>
<li>Calculate test statistic for each simulated dataset<br>
</li>
<li>Determine proportion of simulated test statistics that exceed observed test statistic</li>
</ul>
<aside class="notes">

<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="simulation-studies" class="title-slide slide level1 inverse center" data-background-color="black">
<h1>Simulation Studies</h1>

</section>
<section id="assess-type-i-error-rate" class="slide level2">
<h2>Assess Type I Error Rate</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/table-sims.png"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>2 by 2 by 2 factorial design<br>
</li>
<li>400 trait pairs per scenario<br>
</li>
<li>400 bootstrap samples per trait pair</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="assessing-power" class="slide level2">
<h2>Assessing Power</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/power-curves.svg"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>2 by 2 by 5 factorial design<br>
</li>
<li>3 factors: allele effects difference, allele effects partitioning, and inter-locus distance<br>
</li>
<li>Simulated 400 trait pairs per scenario</li>
<li>400 bootstrap samples per trait pair</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section></section>
<section>
<section id="qtl2pleio-r-package" class="title-slide slide level1 inverse center" data-background-color="black">
<h1><code>qtl2pleio</code> R package</h1>

</section>
<section id="qtl2pleio-r-package-1" class="slide level2">
<h2><code>qtl2pleio</code> R package</h2>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/qtl2pleio-github-screenshot.png"></p>
</div>
</div>
<aside class="notes">
<ul>
<li><p>Functions for <span class="math inline">\(d\)</span>-variate, <span class="math inline">\(d\)</span>-QTL scan &amp; profile LOD plots</p></li>
<li><p>Uses C++ for matrix calculations (via Rcpp and RcppEigen)</p></li>
<li><p>Uses <code>gemma2</code> R implementation of GEMMA EM algorithm for multivariate random effects</p></li>
<li><p>Unit tests, vignettes, and version control</p></li>
<li><p>Open source development on GitHub</p></li>
<li><p>unit tests to ensure accurate calculations</p></li>
<li><p>vignettes on analyzing a pair of traits &amp; on cluster computing for bootstrap test</p></li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="qtl2pleio-r-package-2" class="slide level2">
<h2><code>qtl2pleio</code> R package</h2>
<ul>
<li><code>qtl2pleio</code> R package: <a href="https://fboehm.us/software/qtl2pleio/" class="uri">https://fboehm.us/software/qtl2pleio/</a></li>
</ul>
<div class="cell">
<div class="cell-output-display">
<p><img data-src="figs/qtl2pleio-pkgdown-site-screenshot.png"></p>
</div>
</div>
<aside class="notes">
<ul>
<li>Also available on CRAN</li>
</ul>
<style type="text/css">
        span.MJX_Assistive_MathML {
          position:absolute!important;
          clip: rect(1px, 1px, 1px, 1px);
          padding: 1px 0 0 0!important;
          border: 0!important;
          height: 1px!important;
          width: 1px!important;
          overflow: hidden!important;
          display:block!important;
      }</style></aside>
</section>
<section id="thank-you" class="slide level2">
<h2>Thank you!</h2>
</section>
<section id="references" class="slide level2 smaller scrollable" data-visibility="uncounted">
<h2>References</h2>
<p><img src="figs/vitruvian-mouse-eleni.png" class="slide-logo"></p>
<div class="footer footer-default">
<p>Frederick Boehm (frederick.boehm@gmail.com) https://fboehm.us/</p>
</div>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-jiang1995multiple" class="csl-entry" role="doc-biblioentry">
Jiang, Changjian, and Zhao-Bang Zeng. 1995. <span>“Multiple Trait Analysis of Genetic Mapping for Quantitative Trait Loci.”</span> <em>Genetics</em> 140 (3): 1111–27.
</div>
<div id="ref-logan2013high" class="csl-entry" role="doc-biblioentry">
Logan, Ryan W, Raymond F Robledo, Jill M Recla, Vivek M Philip, Jason A Bubier, Jeremy J Jay, Carter Harwood, et al. 2013. <span>“High-Precision Genetic Mapping of Behavioral Traits in the Diversity Outbred Mouse Population.”</span> <em>Genes, Brain and Behavior</em> 12 (4): 424–37.
</div>
<div id="ref-recla2014precise" class="csl-entry" role="doc-biblioentry">
Recla, Jill M, Raymond F Robledo, Daniel M Gatti, Carol J Bult, Gary A Churchill, and Elissa J Chesler. 2014. <span>“Precise Genetic Mapping and Integrative Bioinformatics in Diversity Outbred Mice Reveals Hydin as a Novel Pain Gene.”</span> <em>Mammalian Genome</em> 25 (5-6): 211–22.
</div>
</div>
</section></section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="utsw-mouse-slides_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/reveal-chalkboard/plugin.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/quarto-tone/tone.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="utsw-mouse-slides_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleChalkboard(event)\"><kbd>b</kbd> Toggle Chalkboard</a></li>\n<li class=\"slide-tool-item\" data-item=\"5\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.toggleNotesCanvas(event)\"><kbd>c</kbd> Toggle Notes Canvas</a></li>\n<li class=\"slide-tool-item\" data-item=\"6\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.downloadDrawings(event)\"><kbd>d</kbd> Download Drawings</a></li>\n<li class=\"slide-tool-item\" data-item=\"7\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'chalkboard': {"buttons":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: 'c/t',

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'slide',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, RevealChalkboard, QuartoTone, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        target: function(trigger) {
          return trigger.previousElementSibling;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'quarto-reveal',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>